libs/compute_api/src/spec.rs

```diff
@@ -252,6 +262,17 @@ pub enum ComputeMode {
     Replica,
 }
 
+/// Log level for audit logging
+/// Disabled, log, hipaa
+/// Default is Disabled
+#[derive(Clone, Debug, Default, Eq, PartialEq, Deserialize, Serialize)]
+pub enum ComputeAudit {
+    #[default]
+    Disabled,
```

Comment:
NIT: Can you make it so that in json this enum was in lowercase? It's kinda standard, and Uppercase in REST API look off to me

`[#[serde(rename_all = "snake_case")]](https://serde.rs/container-attrs.html#rename_all)` will probably do the trick

Reply:
Uh, I actually changed it to be similar to the ComputeMode.
Will leave as is, because [cloud code](https://github.com/neondatabase/cloud/blob/main/goapp/controlplane/apispec/compute-v2.yaml#L320) already has uppercase names in this enum.